distribution/kernel/carbon-home/repository/resources/conf/default.json

```diff
@@ -2,6 +2,7 @@
   "server.default_cache_timeout": "15m",
   "server.force_local_cache": false,
   "server.enableMTOM": "optional",
+  "server.file_upload_config.total_file_size_limit": "100",
```

Comment:
how about server.file_upload.file_size_limit?

Reply:
changed to server.file_upload.file_size_limit